src/main/java/com/worldwarofants/game/module/newgame/NewGameView.java

```diff
@@ -12,12 +12,16 @@ public NewGameView(NewGameViewModel viewModel) {
 	public void renderNewGameScreen() {
 		String startGamePrompt = String.format("* Start Game - %s ", NewGameCommandHandler.COMMAND_START_NEW_GAME);
 		String backPrompt = String.format("* Back - %s ", NewGameCommandHandler.COMMAND_BACK);
-		ConsoleOutputHandler.lineBreak();
-		ConsoleOutputHandler.spell("-- GAME SETUP --");
-		ConsoleOutputHandler.lineBreak();
-		ConsoleOutputHandler.post(startGamePrompt);
-		ConsoleOutputHandler.post(backPrompt);
-		ConsoleOutputHandler.lineBreak();
-		ConsoleOutputHandler.post("Please enter your command:");
+
+		ConsoleOutputHandler console = new ConsoleOutputHandler();
```

Comment:
Can you try to make it so that you don't have to write the keyword `new`? And to be able to just say ConsoleOutputHandler.newOutput().something().something()

Reply:
so yeah, this is what I meant when I said I know some of this stuff I did was kind of a work around haha. I'm yet to fully get my head around static and non-static in Java. I cannot seem to do what you suggest without making the newOutput() method static. However, if I do make it static then I can no longer use ConsoleOutputhundler as a return value in it. This was the only way I could find to get it to work...

Reply:
There's nothing wrong with making the method static. That is what we have to do, to get that fluent interface. The starting method has to be static, otherwise it can't be called without making an object.

What is stopping you from using ConsoleOutputhundler as a return value?

Reply:
As soon as I make the newOutput() method static it comes up with an error on the return value saying it cannot be referenced. I think I've fixed it though by creating a new ConsoleOutputHandler inside the method.

Reply:
Yup, that's the goal!